### PR TITLE
TA-1309: Test implementation

### PR DESCRIPTION
**Pull request by EvgeniiNikonets**

### CODE REVIEW

File: test_files/rate_limit_Include_parameters.json [hunk 1]
```diff
@@ -0,0 +1,431 @@
+{
```
Comment (olgasobolevsky) [inline test_files/rate_limit_Include_parameters.json:1]: This test can be rearranged to save on publish waits.
1. Create rate limit rl1 with 1 tag in Include: ip:123-123-123-123
2. Create rate limit rl2 with 2 tags in  Include: gf1 and gf2
3. Create global filter as described in Step2
4. Create security policy name:"test-sp" with 2 additional path mappings:
a. id: rl1; name: rl1; match:/rl1; limits:["rl1"]
a. id: rl2; name: rl2; match:/rl2; limits:["rl2"]
5. Create server group with this security policy
6. Publish
7. Send 2 requests to path/rl1 with header "x-forwarded-for:123.123.123.123"
8. Send 2 requests to path/rl1 with header "x-forwarded-for:123.123.123.124"
9. Send 2 requests to path/rl2 with header "test:123"
10. Check logs

File: test_files/rate_limit_Include_parameters.json [hunk 2]
```diff
@@ -0,0 +1,473 @@
...
+        {
+          "ID": "wait-for-publish",
+          "Skip": false
+        },
+        {
+          "Name": "Send Traffic using GET method to path /test 1",
+          "Skip": false,
+          "API": {
+            "Base": "send-traffic",
+            "Path": "/rl1"
```
Comment (olgasobolevsky) [inline test_files/rate_limit_Include_parameters.json:279]: In this step the traffic is to path "/rl1" , so the name of the step should be changed to: "Send Traffic using GET method to path /rl1 1".
The same for all Send traffic steps.

File: test_files/rate_limit_Include_parameters.json [hunk 3]
```diff
@@ -0,0 +1,473 @@
...
+            "Text": "b''"
+          }
+        },
+        {
+          "ID": "wait-for-logs",
+          "Skip": false
+        },
+        {
+          "Name": "Assert logs for tag 'limit-name:rl1'",
+          "Skip": false,
```
Comment (olgasobolevsky) [inline test_files/rate_limit_Include_parameters.json:430]: In this step "Assert logs for tag 'limit-name:rl1', so the  Expected result should be
 "SingleResult": [
              {
                "Key": "data/results/0/tags/*",
                "Value": "limit-name:rl1"
              }
            ]

Comment (EvgeniiNikonets) [reply test_files/rate_limit_Include_parameters.json:430]: done

File: test_files/rate_limit_Include_parameters.json (hunk 2 above)
Comment (EvgeniiNikonets) [reply test_files/rate_limit_Include_parameters.json:279]: done

File: test_files/rate_limit_Include_parameters.json [hunk 4]
```diff
@@ -0,0 +1,515 @@
...
+              {
+                "Key": "Server",
+                "Value": "rhino-core-shield"
+              }
+            ],
+            "SingleResult": [
+              {
+                "Key": "data/results/1/tags/*",
+                "Value": "limit-name:rl1"
+              }
```
Comment (olgasobolevsky) [inline test_files/rate_limit_Include_parameters.json:508]: When you filter by path /rl1 and status=471, you get requests only for rate limit rl1. So, the json path should be: "data/results/0/tags/*" .
(with jsonpath "data/results/1/tags/*", we get "Out of range" error)

**Review by olgasobolevsky — APPROVED** [review]

**Pull request merged**